docs/domains/Domain.json

```diff
@@ -272,27 +268,27 @@
           }
         },
         "imagePullSecrets": {
-          "description": "A list of image pull secrets for the WebLogic Docker image.",
+          "description": "A list of image pull secrets for the WebLogic container image.",
           "type": "array",
           "items": {
             "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.LocalObjectReference"
           }
         },
         "domainUID": {
           "pattern": "^[a-z0-9-.]{1,253}$",
-          "description": "Domain unique identifier. Must be unique across the Kubernetes cluster. Not required. Defaults to the value of metadata.name.",
+          "description": "Domain unique identifier. Must be unique across the Kubernetes cluster. This value is distinct and need not match the domain name from the WebLogic domain configuration. Not required. Defaults to the value of metadata.name.",
```

Comment:
I thought the UID uniqueness requirement is more relaxed, e.g. per Operator, or even per namespace?